docs/scos/dev/feature-integration-guides/202200.0/glue-api/glue-api-checkout-feature-integration.md

```diff
@@ -0,0 +1,791 @@
+| PaymentMethodTransfer                                | class    | created    | src/Generated/Shared/Transfer/PaymentMethodTransfer.php      |
+| QuoteTransfer                                        | class    | created    | src/Generated/Shared/Transfer/QuoteTransfer.php              |
+| StoreTransfer                                        | class    | created    | src/Generated/Shared/Transfer/StoreTransfer.php              |
+| MoneyValueTransfer                                   | class    | created    | src/Generated/Shared/Transfer/MoneyValueTransfer.php         |
+| CurrencyTransfer                                     | class    | created    | src/Generated/Shared/Transfer/CurrencyTransfer.php           |
+| QuoteResponseTransfer                                | class    | created    | src/Generated/Shared/Transfer/QuoteResponseTransfer.php      |
+| QuoteErrorTransfer                                   | class    | created    | src/Generated/Shared/Transfer/QuoteErrorTransfer.php         |
+| ShipmentTransfer                                     | class    | created    | src/Generated/Shared/Transfer/ShipmentTransfer.php           |
+| RestErrorCollectionTransfer                          | class    | created    | src/Generated/Shared/Transfer/RestErrorCollectionTransfer.php |
+| CheckoutDataTransfer                                 | class    | created    | src/Generated/Shared/Transfer/CheckoutDataTransfer.php       |
+| ItemTransfer                                         | class    | created    | src/Generated/Shared/Transfer/ItemTransfer.php               |
+| RestCheckoutResponseTransfer                         | class    | created    | src/Generated/Shared/Transfer/RestCheckoutResponseTransfer.php |
+| RestErrorMessageTransfer                             | class    | created    | src/Generated/Shared/Transfer/RestErrorMessageTransfer.php   |
+| RestCheckoutDataTransfer.quote                       | property | added      | src/Generated/Shared/Transfer/RestCheckoutDataTransfer.php   |
+| RestCheckoutResponseTransfer.checkoutData            | property | added      | src/Generated/Shared/Transfer/RestCheckoutResponseTransfer.php |
+| CheckoutDataTransfer.quote                           | property | added      | src/Generated/Shared/Transfer/CheckoutDataTransfer.php       |
+| RestCheckoutDataResponseAttributesTransfer.addresses | property | deprecated | src/Generated/Shared/Transfer/RestCheckoutDataResponseAttributesTransfer.php |
+| QuoteTransfer.salesOrderThresholdValues              | property | added      | src/Generated/Shared/Transfer/QuoteTransfer.php |
+| RestCartsThresholdsTransfer.deltaWithSubtotal        | property | added      | src/Generated/Shared/Transfer/RestCartsThresholdsTransfer.php |
+| RestCartsThresholdsTransfer.message                  | property | added      | src/Generated/Shared/Transfer/RestCartsThresholdsTransfer.php |
```

Comment:
```
SalesOrderThresholdType
SalesOrderThresholdValue
RestCartsAttributes.thresholds
```
these transfers are not mentioned
probably whole `RestCartsThresholds` should be mentioned, since other properties were not mentioned before as well